setup.py

```diff
@@ -174,7 +179,7 @@
         "assertpy==1.1",
         "pip-tools",
         "pybindgen",
-        "types-protobuf",
+        "types-protobuf<4",
```

Comment:
@danielin917 why is this necessary? the error message in the failing unit tests is `ERROR: Cannot uninstall types-protobuf 4.21.0.3, RECORD file not found. You might be able to recover from this via: 'pip install --force-reinstall --no-deps types-protobuf==4.21.0.3'.`, so this line seems very suspicious to me

Reply:
Hey yea, mentioned above that I pinned this type because the linter was producing a lot of errors using the new version `Library stubs not installed for "google.*"`. One thing that might be causing this is that we have:
 "mypy-protobuf==3.1",